src/main/java/gregtech/api/items/metaitem/MetaOreDictItem.java

```diff
@@ -71,21 +90,33 @@ public void registerModels() {
         }
     }
 
-    public static class OreDictValueItem {
+    public OreDictValueItem addOreDictItem(int id, String materialName, int rgb, MaterialIconSet materialIconSet, OrePrefix orePrefix) {
+        return this.addOreDictItem(id, materialName, rgb, materialIconSet, orePrefix, null);
+    }
+
+    public OreDictValueItem addOreDictItem(int id, String materialName, int materialRGB, MaterialIconSet materialIconSet, OrePrefix orePrefix, String chemicalFormula) {
+        return new OreDictValueItem(id, materialName, materialRGB, materialIconSet, orePrefix, chemicalFormula);
+    }
+
+    public class OreDictValueItem {
 
         private final String materialName;
         private final int materialRGB;
         private final MaterialIconSet materialIconSet;
         private final short id;
         private final OrePrefix orePrefix;
 
-        public OreDictValueItem(short id, String materialName, int materialRGB, MaterialIconSet materialIconSet, OrePrefix orePrefix) {
-            this.id = id;
+        protected String chemicalFormula;
+
+        public OreDictValueItem(int id, String materialName, int materialRGB, MaterialIconSet materialIconSet, OrePrefix orePrefix, String chemicalFormula) {
```

Comment:
Why change this to `int`? Your `addOreDictItem` calls get passed a short, so you can just change those method to accept the short that they are passed instead of an int